scripts/bump.sh

```diff
@@ -21,9 +22,13 @@ log_warning () { log_any "33" "~" "$1"; }
 log_hint    () { log_any "34" "?" "$1"; }
 log_info    () { log_any "35" "i" "$1"; }
 
+break_down_version () {
```

Comment:
no idea what this does in `bash` :thinking:

Reply:
What do you mean? it actually works and you're the one that give me that piece of code a few months ago :laughing:

Reply:
:eyes: 

> it actually works
>
never a good reason not to give an example for some obscur `bash` oneline right? :smirk: 

> and you're the one that give me that piece of code a few months ago
>
:grimacing: 
do not remember that and i do not use `bash` anymore, so...

Reply:
> never a good reason not to give an example for some obscur bash oneline right? 

Right but actually that's not what you asked in the first time. It's simply a find/replace like you would do with `sed` or in `vim`:
```
V=1.2.3
echo ${V//./ } # 1 2 3
```

>  do not remember that and i do not use bash anymore, so...

Doesn't mean you forgot everything, does it?

Reply:
> as it's not a sed command nor a vim command, i'd like to have the example you gave in the docstring of break_down_version 😋

I can do that!
However maybe not tonight, could you approve if there isn't anything else on your mind? to avoid delaying this PR 24h more ?

Reply:
well i dunno, maybe i'll have something to say after the change :shrug:

Reply:
```suggestion
break_down_version () {
# V=1.2.3
# echo ${V//./ } # 1 2 3
``` 
something like that would be sufficient?

Reply:
i'd just add
```
# # Example
# <your example>
```
but otherwise that's good :ok_hand:

Reply:
```suggestion
break_down_version () {
# # Exemple
# V=1.2.3
# echo ${V//./ } # 1 2 3
```

Reply:
actually when I clicked "Commit suggestion" it automatically resolved the thread tell me if it's ok @amtoine

Reply:
not sure about the indentation but eh that's fine :+1:

Reply:
euh yeah you're right, i'll fix it sometime 🙂